docs/tutorials/04-include-in-your-build-tool.md

```diff
@@ -114,6 +114,7 @@ destImageRef, err := alltransports.ParseImageName("docker.io/myusername/my-image
 		SystemContext: &types.SystemContext{
 			DockerAuthConfig: &types.DockerAuthConfig{
 				Username: "your username",
+        Password: "your password",
```

Comment:
This is a possible nit of a nit, but there seems to be a space vs tab issue on this particular line.